src/server/pfs/server/commit_set.go

```diff
@@ -0,0 +1,497 @@
+package server
+
+import (
+	"context"
+	"sort"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/pachyderm/pachyderm/v2/src/client"
+	col "github.com/pachyderm/pachyderm/v2/src/internal/collection"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pfsdb"
+	"github.com/pachyderm/pachyderm/v2/src/internal/transactionenv/txncontext"
+	"github.com/pachyderm/pachyderm/v2/src/pfs"
+	pfsserver "github.com/pachyderm/pachyderm/v2/src/server/pfs"
+)
+
+// returns CommitInfos in a commit set, topologically sorted.
+// A commit set will include all the commits that were created across repos for a run, along
+// with all of the commits that the run's commit's rely on (present in previous commit sets).
+func (d *driver) inspectCommitSetImmediateTx(txnCtx *txncontext.TransactionContext, commitset *pfs.CommitSet, filterAliases bool) ([]*pfs.CommitInfo, error) {
```

Comment:
nit, commitset -> commitSet. That naming convention should hold elsewhere as well.